AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_ReflectionProbeAdded.py

```diff
@@ -154,44 +202,185 @@ def AtomEditorComponents_ReflectionProbe_AddedToEntity():
                 TestHelper.wait_for_condition(lambda: not reflection_probe_entity.has_component(shape), 1.0)
                 Report.result(Tests.reflection_probe_disabled, not reflection_probe_component.is_enabled())
 
-        # 8. Enter/Exit game mode.
+        # 8. Toggle Parallax Correction (default True)
+        reflection_probe_component.set_component_property_value(
+            AtomComponentProperties.reflection_probe('Parallax Correction'), False)
+        reflection_probe_component.get_component_property_value(
```

Comment:
Looks like this get isn't being used for anything (it's called again immediately afterwards in the test).

Reply:
good catch. I'll remove that.